roles/keystone/tasks/federation.yml

```diff
@@ -10,8 +10,10 @@
   retries: 5
 
 - name: download mod_auth_openidc
-  get_url: url={{ keystone.federation.sp.oidc.download.url }}
-           dest=/tmp/{{ keystone.federation.sp.oidc.module_name }}
+  get_url:
+    url: "{{ keystone.federation.sp.oidc.download.url }}"
+    dest: /tmp/{{ keystone.federation.sp.oidc.module_name }}
+    checksum: "{{ keystone.federation.sp.oidc.download.checksum }}"
```

Comment:
So I noticed that the apt module's `deb` argument can take a full URL. Is there a reason why we're downloading first and then installing?